layouts/index.html

```diff
@@ -191,16 +191,16 @@ <h3 class="mt-6 text-white">Install with Conda</h3>
                     <br>
                     <p><strong>2.</strong> Then quick install RAPIDS with:</p>
                     <pre
-                        class="highlight use-white-copy"><code>conda create -n rapids-24.12 -c rapidsai -c conda-forge -c nvidia rapids=24.12 python=3.12 cuda-version=12.5</code></pre>
+                        class="highlight use-white-copy"><code>conda create -n rapids-25.02 -c rapidsai -c conda-forge -c nvidia rapids=25.02 python=3.12 cuda-version=12.8</code></pre>
 
                     <h3 class="mt-6 text-white">Install with pip</h3>
                     <p>Install via the NVIDIA PyPI index:</p>
                     <pre class="highlight use-white-copy"><code>pip install \
   --extra-index-url=https://pypi.nvidia.com \
-  cudf-cu12==24.12.* \
-  dask-cudf-cu12==24.12.* \
-  cuml-cu12==24.12.* \
-  cugraph-cu12==24.12.*</code></pre>
+  cudf-cu12==25.02.* \
```

Comment:
@raydouglass @AyodeAwe FYI, we typically use PEP440 tags for these values, like 25.2 instead of 25.02. That’s how our PyPI packages are actually versioned. I’m not able to file a fix right now but maybe someone else can?

Reply:
fixed in https://github.com/rapidsai/rapids.ai/pull/433